### ISSUE TITLE
Fill in additional `people` and `more_people` data fields

### ISSUE THREAD

**Issue by geealbers** [thegetty/scores#6]
As discussed, we're going to add `people` and `more_people` fields to the Scores data to use in filtering for the Image Index. 

`people` should be those expressly mentioned in the `title`, `maker`, or `extended_caption` fields; or those who should have been mentioned there (ie., for a Tudor transcription of a Cage work, the `people` should be both Cage and Tudor even if only one is mentioned currently); or those who show up in print in the actual image that are central to the Scores Project (ie., Feldman mentioned in a program that is otherwise focused on Bussotti). 

`more_people` should be the wider cast of characters that show up in print in the actual image, that are less central to the Scores Project (ie., the names of the participants in a Shiomi or Kaprow piece).

We may ultimately merge the two lists, but we're keeping them separate for now until we see how many end up being listed in the Image Index filter, and how manageable and useful that list will be for readers. 

Names should be listed in "Lastname, Firstname" order. Please check with Editorial if you are unsure of the ordering where full names include three or more parts. 

Like with the `year_filter` and `maker` attributes, the `people` and `more_people` fields support lists and so should be formatted as:

```
people: [ "Lastname, Firstname", "Lastname, Firstname" ]
```
I've gone through and used some automation to copy over the existing names listed in the `maker` field, into the new `people` field in our preferred "Lastname, Firstname" order. Everything should be done except for the makers who we only know by initials, which I queried the team on. If you see an error in these though, go ahead and fix it. I did some spot checking but the automation might not have worked perfectly for each instance.

**Comment by geealbers** [comment]
@alexhallenbeck, I've got a new version of the site at https://third-pages--scores-project.netlify.app/image-index/ with the People and More People filters set up, and the images sorted by date. It's looking/working great. With the people, there are (not surprisingly given the number of people you were dealing with) just a few that need some clean up. For example, there are two versions of Merce Cunningham, a Ben and and a Benjamin Paterson, etc. Can you use the dropdowns to do a proofreading pass and update the objects.yaml file accordingly to remove the erroneous duplicates?

**Comment by alexhallenbeck** [comment]
Yep, no problem! I figured this would happen and am happy to go through and normalize. I’ll send a PR when I’m done (probably first thing tomorrow morning).

From: Greg Albers ***@***.***>
Date: Wednesday, October 30, 2024 at 1:26 PM
To: thegetty/scores ***@***.***>
Cc: Alex Hallenbeck ***@***.***>, Mention ***@***.***>
Subject: Re: [thegetty/scores] Fill in additional `people` and `more_people` data fields (Issue #6)

@alexhallenbeck<https://github.com/alexhallenbeck>, I've got a new version of the site at https://third-pages--scores-project.netlify.app/image-index/ with the People and More People filters set up, and the images sorted by date. It's looking/working great. With the people, there are (not surprisingly given the number of people you were dealing with) just a few that need some clean up. For example, there are two versions of Merce Cunningham, a Ben and and a Benjamin Paterson, etc. Can you use the dropdowns to do a proofreading pass and update the objects.yaml file accordingly to remove the erroneous duplicates?
